File: Practice/Beginner/Enormous Input Test/Enormous-Input-Test.py
""" 
    AUTHOR  -  Atharva Deshpande
    GITHUB PROFILE -  https://github.com/AtharvaD11
    QUESTION LINK  -  https://www.codechef.com/problems/INTEST
"""


(n, k) = map(int, input().split(' '))

ans = 0

for i in range(n):
	x = int(input())
	if x % k == 0:
		ans += 1

print(ans)	

File: Practice/Beginner/Enormous Input Test/C/enormous_input_test.c
#include<stdio.h>
int main()
{
int t,n,k;
scanf("%d",&t);
scanf("%d",&n);
scanf("%d",&k);
int ans=0;
int i=0;
for ( i = 0; i < n; i++)
{
  if (t%k==0)
  {
     ans++;

  }
}
printf("%d\n",ans);
return 0;
}
